dask/tests/test_dot.py

```diff
@@ -75,9 +76,20 @@ def test_dot_graph():
     fn = 'test_dot_graph'
     fns = [fn + ext for ext in ['.png', '.pdf', '.dot']]
     try:
-        dot_graph(dsk, filename=fn)
+        i = dot_graph(dsk, filename=fn)
         assert all(os.path.exists(f) for f in fns)
+        assert isinstance(i, Image)
     finally:
         for f in fns:
             if os.path.exists(f):
                 os.remove(f)
+
+    fn = 'mydask' # default, remove existing files
+    fns = [fn + ext for ext in ['.png', '.pdf', '.dot']]
+    for f in fns:
+        if os.path.exists(f):
+            os.remove(f)
```

Comment:
Block above is to delete the "test_dot_graph.*" files created in the test.

This block is to delete existing "mydask.*" (default) files may be created before the test to check no output files are generated